.pipelines/azdo-ci-build-train.yml

```diff
@@ -77,22 +91,28 @@ stages:
     container: mcr.microsoft.com/mlops/python:latest
     timeoutInMinutes: 0
     steps:
-    - script: |
-        python $(Build.SourcesDirectory)/code/register/register_model.py --build_id $(Build.BuildId) --validate True
-      displayName: 'Check if new model registered'
-      env:
-        SP_APP_SECRET: '$(SP_APP_SECRET)'
+    - task: AzureCLI@1
+      inputs:
+        azureSubscription: '$(WORKSPACE_SVC_CONNECTION)'
+        scriptLocation: inlineScript
+        inlineScript: |
+          set -e # fail on error
+          export SUBSCRIPTION_ID=$(az account show --query id -o tsv)
+          python $(Build.SourcesDirectory)/ml_service/pipelines/verify_train_pipeline.py --build_id $(Build.BuildId) --model_name "$(MODEL_NAME)"
+          echo "##vso[task.setvariable variable=MODEL_VERSION;isOutput=true]$(cat model_version.txt)"
```

Comment:
Please fix this output on this step.  You can [see it in this build](https://aidemos.visualstudio.com/MLOps/_build/results?buildId=3190&view=logs&j=bdf7ed77-c6a0-5b06-edd2-dd86f84ec0e0&t=02aa005f-1b50-573a-e7fc-2dcee0005b34).

```
Model was registered for this build.
cat: model_version.txt: No such file or directory
```

Reply:
This is likely due to me not specifying a MODEL_VERSION variable.  If you set the default value of the variable to be `model_version.txt` then we'll probably be good.

Reply:
Line 102 was actually dead code (belongs in PR #122), I've removed it.

Also updated line 59 (https://github.com/algattik/MLOpsPython/blob/0cf896392717292a66fd8ca02164adecc605e4f1/.pipelines/azdo-ci-build-train.yml#L59) where a similar logic is used, so that the failure of the `cat` command fails the whole job.